script.js

```diff
@@ -86,7 +86,7 @@ function addBookToLibrary() {
 }
 
 function checkCreatedValid() {
-  if (getId('title').value === true) {
+  if (getId('title').value !== '') {
     addBookToLibrary();
   }
 }
```

Comment:
It will be good to validate not only title, but maybe the other fields too

Reply:
We're here to help ourselves makes apps as professional as possible
![Screen Shot 2020-03-21 at 12 03 35 PM](https://user-images.githubusercontent.com/40712459/77225018-7e213680-6b6b-11ea-9b0a-fa2770d3e782.png)